### PR TITLE
Drop resolv.conf hack

### PR DESCRIPTION
**Pull request by k0da**
This commit drops resolv.conf hack and uses POD's dnsConfig instead

Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

**Review by ytsarev — APPROVED** [review]
looks great

File: Makefile [hunk 1]
```diff
@@ -478,9 +478,9 @@ kubectl config use-context $1 > /dev/null
 endef
 
 define hit-testapp-host
-	kubectl run -it --rm busybox --restart=Never --image=busybox -- sh -c \
-	"echo 'nameserver `$(K8GB_COREDNS_IP)`' > /etc/resolv.conf && \
-	wget -qO - $1"
+	kubectl run -it --rm busybox --restart=Never --image=busybox --command \
+	--overrides='{"spec": {"dnsConfig": {"nameservers": ["$(K8GB_COREDNS_IP)"]}, \"dnsPolicy\": \"None\"}}' \
+	-- wget $1
```
Comment (jkremser) [inline Makefile:483]: `-qO -` is missing (we can use curl actually instead) (`-q` ... quiet (no wget stuff), `-O -` print to stdout)

**Review by jkremser — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (k0da) [reply Makefile]: @jkremser good catch! Amended

**Comment by jkremser** [comment]
Fixes https://github.com/k8gb-io/k8gb/issues/712 (nice change)

**Review by jkremser — APPROVED** [review]

**Pull request merged**